webapp/backend/router/ai_operations.py

```diff
@@ -40,23 +40,34 @@ def get_ai_operation( ai_operations_service: Annotated[AIOperationsService, Depe
 
 
 @ai_operations.put("/operation/{id}")
-def update_ai_operations(id: UUID, ai_operations : AIOperationsUpdateModel, ai_operations_service: Annotated[AIOperationsService, Depends(AIOperationsService)], current_user: Annotated[ReadUserModel, Depends(get_current_user)]):
+async def update_ai_operations(id: UUID, ai_operations : AIOperationsUpdateModel, ai_operations_service: Annotated[AIOperationsService, Depends(AIOperationsService)], current_user: Annotated[ReadUserModel, Depends(get_current_user)]):
     if( current_user.role == Role.admin or  current_user.role == Role.moderator):
-        return ai_operations_service.update_ai_operations(id, ai_operations)
+        return await ai_operations_service.update_ai_operations(id, ai_operations)
     raise HTTPException(status_code=401, detail="Unauthorized")
 
 
 
-@ai_operations.get("/start-audio-processsing")
-def start_audio_processing(video_service: Annotated[LiveVideoService, Depends(LiveVideoService)]):
+@ai_operations.get("/start-audio-processing")
+async def start_audio_processing(video_service: Annotated[LiveVideoService, Depends(LiveVideoService)], ai_operations_service: Annotated[AIOperationsService, Depends(AIOperationsService)]):
     try:
         # extract audio from video
         # video_to_process = videos[0]
         # process and send to ML layer
         video = video_service.get_status_live_video(VideoStatus.live)
+        ai_operations = ai_operations_service.get_ai_operation()
+        stages = {
+            "STAGE1": 1,
+            "STAGE2": 2,
+            "STAGE3": 3,
+            "STAGE4": 4,
+            "STAGE5": 5
```

Comment:
I think these should just be called rounds so it's uniform everywhere and to everyone. `ROUND1`, `ROUND2` etc introducing a new terminology will be confusing later

@markowusu please let's update other areas to rounds instead of stages if possible